src/mapgen/mapgen_carpathian.cpp

```diff
@@ -227,6 +227,12 @@ void MapgenCarpathianParams::writeParams(Settings *settings) const
 }
 
 
+void MapgenCarpathianParams::setDefaultSettings(Settings *settings)
```

Comment:
Would these functions be better named `setDefaultFlags()` as that is what they do?

Reply:
It's what they currently do. This function is intended to be used for further settings, such as defaults for noise params.

Reply:
Ok, i suspected these might be extended to other settings. This is fine.